taskmgr/server/server.go

```diff
@@ -57,6 +58,34 @@ func (t TasksHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func NewTaskHandler(w http.ResponseWriter, r *http.Request) {
+	if r.Method != http.MethodPost {
+		logger.ErrorLogger.Printf("Method: %s is not allowed on /newtask endpoint\n", r.Method)
+		w.WriteHeader(http.StatusMethodNotAllowed)
+		return
+	}
+
+	logger.InfoLogger.Printf("Got request on /newtask endpoint\n")
+
+	body, err := io.ReadAll(r.Body)
+	if err != nil {
+		logger.ErrorLogger.Printf("Couldn't read the body")
+	}
+
+	var tempnewtask task.Task
```

Comment:
In go we use CamelCase
```suggestion
	var tempNewTask task.Task
```